dist/dss.js

```diff
@@ -1,53 +1,56 @@
 //File : src/dss.js
 
-
-(function(world){
+(function(world) {
   'use strict';
 
   /**
    * [dss description]
    * @return {[Object]}
    */
-  world.dss = function(){ 
-
+  world.dss = function() {
     return {
       core : {}
     };
-
   }();
-
 })(this);
+
 //File : src/core/dss.core.ajax.js
 
-(function(dss){
+(function(dss) {
 	'use strict';
 
- dss.core.ajax = function(args){
- 	var callback = args.callback || function(){return false;};
+ dss.core.ajax = function(args) {
+ 	var callback = args.callback || function() { return false; };
  	var url = args.url;
  	var method = args.method || 'GET';
  	var data = args.data || false;
-
  	var xhr;
- 	
- 	if(typeof XMLHttpRequest !== 'undefined') xhr = new XMLHttpRequest();
- 	else {
- 		var versions = ["MSXML2.XmlHttp.5.0", 
- 		"MSXML2.XmlHttp.4.0",
- 		"MSXML2.XmlHttp.3.0", 
- 		"MSXML2.XmlHttp.2.0",
- 		"Microsoft.XmlHttp"];
- 		
+
+ 	if (typeof XMLHttpRequest !== 'undefined') {
+    xhr = new XMLHttpRequest();
+  } else {
+ 		var versions = [
+      'MSXML2.XmlHttp.5.0',
```

Comment:
No, actually the original file is good, since it uses only tabs! See the same slice at original file:

<img width="302" alt="captura de tela 2015-09-22 as 20 11 28" src="https://cloud.githubusercontent.com/assets/2164551/10034402/a004b6f0-6166-11e5-9da5-9562191cd7b9.png">

General advice about sending PRs: open an issue before sending a PR because is a kinda intrusive sending a PR without a prior demand. Since this is a newborn project, there's a lot of important things to contribute and waste effort, so I won't help to check this PR anymore.

Let's create some fresh issues , be cool :sunglasses: